scripts/generateCssFlow.js

```diff
@@ -0,0 +1,19 @@
+const path = require('path')
+const fs = require('fs')
+
+const distDir = path.resolve(__dirname, '../dist')
+
+if (!fs.existsSync(distDir)) {
+  throw new Error('dist folder not available')
+}
+
+const files = fs.readdirSync(distDir)
+
+files.forEach(file => {
+  const parsed = path.parse(file)
+  if (parsed.name.endsWith('.css')) {
+    const fullPath = path.resolve(__dirname, '../dist', file)
+    console.log('stubbing css file', fullPath)
+    fs.writeFileSync(fullPath, '')
+  }
+})
```

Comment:
This could be a bit simpler with

```javascript
function isCSSFile(file) {
  return path.parse(file).endsWith('.css') && !fs.lstatSync(file).isDirectory()
}

function getTargetPath(file) {
  return path.resolve(__dirname, '../dist', file)
}

function writeEmptyFile(path) {
  return fs.writeFileSync(path, '')
}

fs
  .readdirSync(distDir)
  .filter(isCSSFile)
  .map(getTargetPath)
  .forEach(writeEmptyFile)
```